src/csharp/Microsoft.Spark.Worker/Command/SqlCommandExecutor.cs

```diff
@@ -737,6 +738,25 @@ internal ArrowOrDataFrameGroupedMapCommandExecutor(Version version)
             return ExecuteArrowGroupedMapCommand(inputStream, outputStream, commands);
         }
 
+        private RecordBatch WrapArrowRecordBatchColumnsInAStruct(RecordBatch batch)
+        {
+            if (SparkEnvironment.SparkVersion >= new Version(Versions.V3_0_0))
```

Comment:
I _think_ the unit tests are failing on `SparkEnvironment.SparkVersion`. Maybe it isn't available during unit tests??

Reply:
Yea I think so too. I just pinged @suhsteve about it. Maybe there's another way to get the version, I'm not sure

Reply:
I commented it out for now to test CI. I'll turn it back on once I hear from @suhsteve or @imback82

Reply:
Can you use `_version` instead ?